src/modules/awake/Awake/Espresso.csproj

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Shouldn't the name of the project be `Awake` or something similar?

Reply:
honestly, i'm fine with this, not end of world, dlls and everything are correct.  other instances of stuff like this.

Reply:
I actually removed this because it's a left-over file.